kong/runloop/handler.lua

```diff
@@ -105,20 +105,20 @@ local set_upstream_ssl_verify_depth
 local set_upstream_ssl_trusted_store
 local set_authority
 local set_log_level
+
+set_upstream_cert_and_key = ktls.set_upstream_cert_and_key
+set_upstream_ssl_verify = ktls.set_upstream_ssl_verify
+set_upstream_ssl_verify_depth = ktls.set_upstream_ssl_verify_depth
+set_upstream_ssl_trusted_store = ktls.set_upstream_ssl_trusted_store
```

Comment:
Now that we're not conditionally setting these vars based on subsystem, I think these lines should just be merged into the `local` declarations on lines 102-104:

```lua
local set_upstream_cert_and_key = ktls.set_upstream_cert_and_key
local set_upstream_ssl_verify = ktls.set_upstream_ssl_verify
local set_upstream_ssl_verify_depth = ktls.set_upstream_ssl_verify_depth
local set_upstream_ssl_trusted_store = ktls.set_upstream_ssl_trusted_store
```